onboarding/serializers.py

```diff
@@ -317,7 +326,7 @@ class WhenPackageSendToEmployeeSerializer(serializers.ModelSerializer):
     class Meta:
         model = PackagesUsers
         fields = (
-            'package',
+            'package'
```

Comment:
Missing `package` results that user can not see the dates when packages was sent.
Missing comma `,` results:
`django.core.exceptions.ImproperlyConfigured: Field name 'packagesend_on' is not valid for model 'PackagesUsers'.`